tests/fsharp/test-framework.fs

```diff
@@ -33,9 +33,8 @@ module Commands =
         Directory.CreateDirectory ( Path.Combine(workDir, dir) ) |> ignore
 
     let rm dir path =
-        log "rm %s" path
         let p = path |> getfullpath dir
-        if File.Exists(p) then File.Delete(p)
+        if File.Exists(p) then (log "rm %s" p) |> ignore; File.Delete(p); else (log "not found: %s p") |> ignore
```

Comment:
Haven't checked the code in VS, but are the ignore required?

let's just put this on separate lines instead of `;`.

I like the use of IDisposable to make it work in a scope.